webapp/tests/bootstrap.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+use Symfony\Component\Dotenv\Dotenv;
+
+require dirname(__DIR__, 2) . '/lib/vendor/autoload.php';
+require dirname(__DIR__) . '/config/load_db_secrets.php';
+
+if (file_exists(dirname(__DIR__) . '/config/bootstrap.php')) {
+    require dirname(__DIR__) . '/config/bootstrap.php';
+} elseif (method_exists(Dotenv::class, 'bootEnv')) {
+    (new Dotenv())->bootEnv(dirname(__DIR__) . '/.env');
+}
+
+if ($_SERVER['APP_DEBUG']) {
+    umask(0000);
```

Comment:
Why this?

Reply:
Ask symfony, it’s part of their current recipe and got auto added. Again, we can remove it but I’d like to stick to upstream as much as possible

Reply:
I would leave this out indeed. It seems weird that we would use a different umask in debug mode.

Reply:
I will check if I can find what their reasoning is to see if it makes sense. Otherwise I will remove it

Reply:
https://github.com/symfony/recipes/commit/57e4d9ee9628aba119a1166390e9c427c6dc00fa no reason given for it. I will drop it.